ws-android/Dockerfile

```diff
@@ -5,65 +5,180 @@ FROM cloud9/workspace
+
+
+
+
+
+
+
+# -------------------------- Apache Ant not installed this docker -----------------------------------------
+# Install Apache-Ant
+#refer to http://www.eu.apache.org/dist//ant/binaries/apache-ant-1.9.6-bin.zip
+#RUN cd /usr/local/ && \
+#    curl -L -O http://ftp.meisei-u.ac.jp/mirror/apache/dist//ant/binaries/apache-ant-1.9.2-bin.tar.gz && \
+#    tar xf apache-ant-1.9.2-bin.tar.gz && \
+#    rm -f apache-ant-1.9.2-bin.tar.gz && \
+#    chown -R ubuntu:ubuntu /usr/local/apache-ant-1.9.2 && \
+#    chmod -R 774 /usr/local/apache-ant-1.9.2
 
+# Environment variables for Apache
+#ENV ANT_HOME /usr/local/apache-ant-1.9.2
+#ENV PATH $PATH:$ANT_HOME/bin
+
+
+
+
+
+
+# -------------------------- Install Gradle -----------------------------------------
 # Install Gradle
-RUN GRADLE_VERSION=2.5 && \
-    cd /usr/local && \
+ENV GRADLE_VERSION 2.5
+
+RUN cd /usr/local && \
     curl -L https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-bin.zip -o gradle-bin.zip && \
     unzip gradle-bin.zip && \
-    rm gradle-bin.zip
-
-# Update & Install Android Tools
-# Cloud message, billing, licensing, play services, admob, analytics
-RUN yes | (\
-    /usr/local/android-sdk-linux/tools/android update sdk --filter tools --no-ui --force -a && \
-    /usr/local/android-sdk-linux/tools/android update sdk --filter platform-tools --no-ui --force -a && \
-    /usr/local/android-sdk-linux/tools/android update sdk --filter android-19 --no-ui --force -a && \
-    /usr/local/android-sdk-linux/tools/android update sdk --filter android-20 --no-ui --force -a && \
-    /usr/local/android-sdk-linux/tools/android update sdk --filter android-21 --no-ui --force -a && \
-    /usr/local/android-sdk-linux/tools/android update sdk --filter android-22 --no-ui --force -a && \
-    /usr/local/android-sdk-linux/tools/android update sdk --filter android-23 --no-ui --force -a && \
-    /usr/local/android-sdk-linux/tools/android update sdk --filter extra --no-ui --force -a && \
-    /usr/local/android-sdk-linux/tools/android update sdk --all --filter build-tools-23.0.2 --no-ui --force) 
-
-# Set PATH
-RUN \
-    echo 'export ANDROID_HOME=/usr/local/android-sdk-linux' > /etc/profile.d/android.sh && \
-    echo 'export ANDROID_SDK_HOME=/usr/local/android-sdk-linux' >> /etc/profile.d/android.sh && \
-    echo 'export ANDROID_NDK_HOME=/usr/local/android-ndk-r10e' >> /etc/profile.d/android.sh && \
-    echo 'export GRADLE_HOME=/usr/local/gradle-2.5' >> /etc/profile.d/android.sh && \
-    echo 'export NDK_PROJECT_PATH=/home/ubuntu/workspace' >> /etc/profile.d/android.sh && \
-    echo 'export PATH=$PATH:$ANDROID_HOME/tools:$ANDROID_NDK_HOME/platform-tools:$ANDROID_NDK_HOME:$GRADLE_HOME/bin' >> /etc/profile.d/android.sh
+    rm gradle-bin.zip  && \
+    chmod -R 774 /usr/local/gradle-${GRADLE_VERSION} && \
+    chown -R ubuntu:ubuntu /usr/local/gradle-${GRADLE_VERSION} 
```

Comment:
Is this needed? Things in /usr/local shouldn't be owned by interactive users normally.